qa-tests-backend/src/main/groovy/objects/Namespace.groovy

```diff
@@ -1,10 +1,13 @@
 package objects
 
+import groovy.transform.CompileStatic
+
+@CompileStatic
 class Namespace {
-    def uid
-    def name
-    def labels
-    def deploymentCount
-    def secretsCount
-    def networkPolicyCount
+    String uid
+    String name
+    Map<String, String> labels
+    List<String> deploymentCount
```

Comment:
This is an odd name for a list of strings. Maybe `deployments` is a better name?

Reply:
+1

Reply:
I also wondered why a list would be names `xxxCount` while investigating CI failures.
Renaming sounds like a sane option to me too.

Reply:
Fixed